### ISSUE TITLE
Course week 1 lesson 5 is missing

### ISSUE THREAD

**Issue by yaxu** [tidalcycles/tidal-doc#273]
https://club.tidalcycles.org/t/week-1-lesson-5-mini-notation-part-3/449/23

Also would be great to have links to the forum throughout, as there's a lot of additional info there.

**Comment by Bodhert** [comment]
Hello! this is something I can collaborate with? it will be adding markdown and a few links

**Comment by cleary** [comment]
> Hello! this is something I can collaborate with? it will be adding markdown and a few links

Yes of course - are you familiar with forking and making Pull Requests on github?

**Comment by Bodhert** [comment]
Hello, I open the fix [here](https://github.com/tidalcycles/tidal-doc/pull/288), let me know if that works, If soo I will add forum links to `course2.mdx`

**Comment by yaxu** [comment]
Looks good, thanks! https://tidalcycles.org/docs/patternlib/tutorials/course1#lesson-5-mini-notation-part-iii
